e2e/cypress/integration/login/login_spec.js

```diff
@@ -9,7 +9,7 @@
 
 let config;
 
-describe('Login page', () => {
+describe('@smoke Login page', () => {
```

Comment:
I wish there's other solution out there to address grouping effectively without changing "title" field.  I'm not sure how the other platform tracked it, but I'm planning to save all test histories so we could build insights on top of it.  One way to uniquely identify a test is thru its title, specifically `fullTitle`.  With mochawesome, test has the following properties:
```json
{
    "title": "monokai theme renders correctly",
    "fullTitle": "AS14319 Theme Colors - Code monokai theme renders correctly",
    "timedOut": null,
    "duration": 25713,
    "state": "passed",
    "speed": "slow",
    "pass": true,
    "fail": false,
    "pending": false,
    "context": null,
    "code": "// code //",
    "err": {},
    "uuid": "00114ce8-bd2c-4fc2-8989-0dc8eaefbb8c",
    "parentUUID": "676e4704-75cb-4e99-863d-42333c23bdb4",
    "isHook": false,
    "skipped": false
},
```

How about manually setting those for `smoke`, `enterprise`, etc. at `e2e/run_tests.js`?

Reply:
I think this is a larger discussion, added to meeting notes.

Reply:
Hey @thekiiingbob - touching base to see if there are next steps?

Reply:
@jasonblais I'm gonna close this PR as we need to rethink our approach to tagging to include test case ID's as well.